include/emulator/codeGCN.h

```diff
@@ -0,0 +1,24 @@
+#ifndef _CODEGCN_H
```

Comment:
hmm is this codeRVL.h now? ditto other GCN.h files (and does mcardGCN exist at all now?)

Reply:
what about mcardGCN?

Reply:
I removed the file, it became unused

Reply:
oh, I think I got tricked by github sorry